sea-orm-codegen/src/entity/writer.rs

```diff
@@ -297,25 +320,83 @@ impl EntityWriter {
             pub use super::#table_name_snake_case_ident::Entity as #table_name_camel_case_ident;
         }
     }
+
+    pub fn gen_compact_model_struct(entity: &Entity) -> TokenStream {
+        let table_name = entity.table_name.as_str();
+        let column_names_snake_case = entity.get_column_names_snake_case();
+        let column_rs_types = entity.get_column_rs_types();
+        let primary_keys: Vec<String> = entity
+            .primary_keys
+            .iter()
+            .map(|pk| pk.name.clone())
+            .collect();
+        let attrs: Vec<TokenStream> = entity
+            .columns
+            .iter()
+            .map(|col| {
+                let mut attrs: Punctuated<_, Comma> = Punctuated::new();
+                if primary_keys.contains(&col.name) {
+                    attrs.push(quote! { primary_key });
+                    if !col.auto_increment {
+                        attrs.push(quote! { auto_increment = false });
+                    }
+                }
+                if let Some(ts) = col.get_col_type_attrs() {
```

Comment:
We forgot one thing. That is, `nullable` when a col_type is involved.
See https://github.com/SeaQL/sea-orm/issues/105#issuecomment-915922635

Reply:
Oh, will fix this